deployments/ansible/roles/collector/README.md

```diff
@@ -253,3 +253,73 @@ after installation/configuration in order for any change to take effect.
 
 - `splunk_otel_auto_instrumentation_enable_metrics` (Linux only): Enable or
   disable exporting Micrometer metrics. (**default**: `false`)
+
+### Auto Instrumentation for .NET on Windows
```

Comment:
@aurbiztondo-splunk ptal

Reply:
Done, LGTM; left a couple of minor comments.